docker/includes/csp.conf

```diff
@@ -0,0 +1,67 @@
+
```

Comment:
this file isn't supposed to be in this repo

Reply:
Right, this needs to be removed. Still testing out to see whether the build will run in KIND with it.